packages/salesforcedx-vscode-apex-debugger/test/vscode-integration/index.ts

```diff
@@ -7,13 +7,16 @@
 
 // tslint:disable-next-line:no-var-requires
 const testRunner = require('@salesforce/salesforcedx-test-utils-vscode/out/src/testrunner');
-
+import { join, normalize } from 'path';
 // You can directly control Mocha options by uncommenting the following lines
 // See https://github.com/mochajs/mocha/wiki/Using-mocha-programmatically#set-options for more info
-testRunner.configure({
-  ui: 'bdd', // the TDD UI is being used in extension.test.ts (suite, test, etc.)
-  useColors: true, // colored output from test results
-  timeout: 360000
-});
+testRunner.configure(
+  {
+    ui: 'bdd', // the TDD UI is being used in extension.test.ts (suite, test, etc.)
+    useColors: true, // colored output from test results
+    timeout: 360000
+  },
+  normalize(join(__dirname, '..', '..', '..'))
```

Comment:
update all the places where we use the testRunner to pass the correct directory for writing the xml results.  Note this is three levels back from the current directory which is out/test/vscode-integration